doc/openshift-guide.md

```diff
@@ -0,0 +1,205 @@
+# Using 3scale API Gateway on Red Hat OpenShift
+
+This tutorial describes how to use the Dockerized version of the 3scale API Gateway that is packaged for easy installation and operation on Red Hat OpenShift V3.
```

Comment:
Let's indicate that is v2 (and only merge in v2 branch). Maybe also call it (APIcast) in parenthesis.